src/services/domain/roles/roles.service.spec.ts

```diff
@@ -0,0 +1,211 @@
+        userID: testData.user.id,
+      });
+
+      expect(res.applications).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            communityID: testData.rolesUser.applications[0].communityID,
+            hubID: testData.rolesUser.applications[0].hubID,
+          }),
+        ])
+      );
+
+      expect(res.organizations).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            organizationID: testData.organization.id,
+          }),
+        ])
+      );
+
+      expect(res.hubs).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            hubID: testData.hub.id,
+          }),
+        ])
+      );
+    });
+
+    it('Should throw exception', async () => {
+      jest
+        .spyOn(userService, 'getUserWithAgent')
+        .mockResolvedValue(testData.user);
+
+      jest
+        .spyOn(hubService, 'getHubOrFail')
+        .mockResolvedValue(testData.hub as any);
+
+      jest
+        .spyOn(challengeSerivce, 'getChallengeOrFail')
+        .mockResolvedValue(testData.challenge as any);
+
+      jest
+        .spyOn(organizationService, 'getOrganizationOrFail')
+        .mockResolvedValue(testData.organization as any);
+
+      jest
+        .spyOn(opportunityService, 'getOpportunityOrFail')
+        .mockResolvedValue(testData.opportunity as any);
+
+      jest
+        .spyOn(applicationService, 'findApplicationsForUser')
+        .mockResolvedValue(testData.applications as any);
+
+      jest
+        .spyOn(applicationService, 'isFinalizedApplication')
+        .mockResolvedValue(false);
+
+      jest
+        .spyOn(applicationService, 'getApplicationState')
+        .mockResolvedValue('new');
+
+      jest.spyOn(communityService, 'isHubCommunity').mockResolvedValue(false);
+
+      await asyncToThrow(
+        rolesService.getUserRoles({
+          userID: testData.user.id,
+        }),
+        RelationshipNotFoundException
+      );
+    });
+  });
+
+  describe('Organization Roles', () => {
+    it('Should get organization roles', async () => {
+      jest
+        .spyOn(organizationService, 'getOrganizationAndAgent')
+        .mockResolvedValue({
+          organization: testData.organization as any,
+          agent: testData.agent,
+        });
+
+      jest
+        .spyOn(hubService, 'getHubOrFail')
+        .mockResolvedValue(testData.hub as any);
+
+      jest
+        .spyOn(challengeSerivce, 'getChallengeOrFail')
+        .mockResolvedValue(testData.challenge as any);
+
+      jest
+        .spyOn(organizationService, 'getOrganizationOrFail')
+        .mockResolvedValue(testData.organization as any);
+
+      jest
+        .spyOn(opportunityService, 'getOpportunityOrFail')
+        .mockResolvedValue(testData.opportunity as any);
+
+      const res = await rolesService.getOrganizationRoles({
+        organizationID: testData.organization.id,
+      });
+
+      expect(res.hubs).toEqual(
```

Comment:
I am confused. The test should be testing organization roles, but you are asserting hubs.